packages/shared/src/nomination-pools.ts

```diff
@@ -886,6 +886,8 @@ async function nominationPoolGlobalConfigTest<
 
     await relayClient.dev.newBlock()
 
+    // Because this extrinsic was executed via the scheduler technique, its events won't be available
+    // through `checkEvents` - hence the need for this event extraction process.
```

Comment:
there is `checkSystemEvents` can be used

Reply:
I will correct this, and push a fix posthaste.

Reply:
On closer inspection, `checkSystemEvents` here would help in knowing that the manual `scheduler` injection succeeded, but it still wouldn't allow us to check `nominationPool` events (that the extrinsic in question doesn't emit yet, but the previous holds without loss of generality).

However, I see now that there are a few places where `checkSystemEvents` would be helpful - it can be done in a follow-up PR.